From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Thu, 13 Jun 2024 21:32:02 -0500
Subject: [PATCH] fix dev build

This is an optional convenience patch that allows you to build the zed package with `--profile dev` which is much, **much** faster than the release build.

This is useful when you want to experiment with changes and iterate quickly without waiting forever for each build. Without this commit zed will fail to launch as it needs resource files embedded in the binary that are skipped for debug/dev builds.

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index cb58423a8..72dd44a38 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -324,7 +324,7 @@ refineable = { path = "./crates/refineable" }
 regex = "1.5"
 repair_json = "0.1.0"
 rusqlite = { version = "0.29.0", features = ["blob", "array", "modern_sqlite"] }
-rust-embed = { version = "8.4", features = ["include-exclude"] }
+rust-embed = { version = "8.4", features = ["include-exclude", "debug-embed"] }
 schemars = "0.8"
 semver = "1.0"
 serde = { version = "1.0", features = ["derive", "rc"] }
